### ISSUE TITLE
studio code location ?

### ISSUE THREAD

**Issue by JR-Utily** [game-icons/icons#787]
Hello !
Where is the studio code ?
I would be glad to help designing new feature for it, but I can't find the code in the repo...

**Comment by aloisdg** [comment]
I dont now if the studio is open source. I would love it to be though. Maybe in a different project under the game-icons org.

**Comment by KrazyBomb** [comment]
Agreed! It'd be cool to see new features like adding icons to eachother. (So you can combine them in different ways to make unique icons you wouldn't be able to otherwise)
